affiliated/affiliated_package_review_guidelines.md

```diff
@@ -106,7 +106,11 @@ it exists) and see how the package fares:
 </tr>
 <tr>
 <td><img src="https://img.shields.io/badge/Green-brightgreen.svg" alt="Green"></td>
-<td>Extensive documentation, including at least: motivation/scope of package, installation instructions, usage examples, API documentation. In terms of infrastructure, the documentation should be automatically built on readthedocs.org. If appropriate, one or more tutorials should be included in the Astropy tutorials at http://tutorials.astropy.org.</td>
+<td>Extensive documentation, including at least: motivation/scope of package, installation instructions, 
+usage examples, and API documentation. 
+A preferred method of citation of this package and citations to relevant papers and software included where appropriate. 
```

Comment:
Is there a guideline somewhere else on "preferred method of citation"? Even coordinated packages do not qualify under this new requirement. We should clean up our own stuff first before we impose it on others.

Reply:
Citation instructions should be in a CITATION file at the top most level of a repo. I just did a quick spot check and astroquery, photutils, and ccdproc all have these files.  

I will add more words to the [webpage PR](https://github.com/astropy/astropy.github.com/pull/460) with this detail.

Reply:
Thanks for checking but we have more coordinated packages than that. Here are the ones that do not:

* https://github.com/astropy/astropy-healpix
* https://github.com/astropy/regions
* https://github.com/astropy/reproject

Reply:
Also, yes, CITATION file should be low-bar enough. I was worried you were also requiring Zenodo DOI, which while is a good practice, might present higher barrier.

Reply:
tl;dr -- We should add CITATION files to the three coordinated packages above before merging this and astropy/astropy.github.com#460 .